libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -146,6 +155,15 @@ static IotMqttError_t _subscriptionCommon( IotMqttOperationType_t operation,
                                            const IotMqttCallbackInfo_t * pCallbackInfo,
                                            IotMqttOperation_t * pOperationReference );
 
+/**
+ * @brief The timer query function provided to the MQTT context.
+ *
+ * This function returns the elapsed time.
+ *
+ * @return Time in milliseconds.
+ */
+static uint32_t getTimeMs( void );
```

Comment:
As per the discussion yesterday, you were mentioning that the getTimeMs is not required for Phase2. Can this be removed then?